.gitignore

```diff
@@ -21,3 +21,5 @@ fabfile/deploy.py
 htmlcov/
 
 node_modules/
+
+huxley/www/static/js/
```

Comment:
Make this more specific, just in case we throw more JS files in there that we actually want to track.